components/Header.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  AppBar,
+  Box,
+  Container,
+  IconButton,
+  Toolbar,
+  Tooltip,
+  Typography,
+  useTheme,
+} from '@mui/material'
+import GitHubIcon from '@mui/icons-material/GitHub'
+import Image from 'next/image'
+
+const Header = () => {
+  const theme = useTheme()
+  return (
+    <AppBar position="sticky" elevation={0}>
+      <Container maxWidth="xl">
+        <Toolbar disableGutters>
+          <Image src="/zigen.webp" height={60} width={60} alt="logo" />
+          <Typography variant="h5" className="font-bold">
+            <span style={{ color: theme.palette.secondary.main }}>Z</span>
+            <span style={{ color: theme.palette.background.default }}>
+              IGEN
```

Comment:
[As I recall](https://docs.google.com/presentation/d/1zm2GdLfLprfj4wE93c1qMr0iu6qtft7tB6Zl98vq-ak/edit#slide=id.p), in the logotype (font within the logo), the font weight was 900.

Or if you specify the font style in CSS for every single font, you lose the uniformity of the logotype, so I think it is better to make a Z + IGEN svg and embed that in the site.

Reply:
I think the logotype should be text so that it can be readable for machines. How about creating a react component of the logotype?

Reply:
> I think the logotype should be text so that it can be readable for machines.

Certainly, text may be more appropriate for machines like search engines than svg. (However, svg has title tag and desc tag, and for search engines, the project name written in these tags may be no different from the text.)

I'm thinking like the above, but it looks good that you'll create a react component of the logotype!!